src/Stratis.Features.Collateral/CheckCollateralFullValidationRule.cs

```diff
@@ -70,51 +63,6 @@ public override Task RunAsync(RuleContext context)
 
             this.Logger.LogDebug("Commitment is: {0}. Magic is: {1}", commitmentHeight, commitmentNetworkMagic);
 
-            // Strategy:
```

Comment:
When resyncing the Cirrus chain we will always have the scenario where the earlier collateral commitment heights in the chain are from the Stratis era. So I don't think we can simply remove the code. The if will still be entered which hints at the code still being required. Maybe you will get away with replacing it with an InIBD check,